### ISSUE TITLE
Feature Request : Terminal Directive Legality

### ISSUE THREAD

**Issue by ANR-PeterPan** [Null-Signal-Games/netrunnerdb#106]
Can we have a Terminal Directive Legality?
- no MWL
- only 1 Core and TD-Cards
- 3 cards less for both sides
- 6 AP less for Corp
perhaps automaticly switching collection to only 1 Core and Terminal Directive?

**Comment by alsciende** [comment]
This kind of legality is not supported yet. Those are the deck-building rules for TD? So, decks built for TD play are not legal decks?

**Comment by ANR-PeterPan** [comment]
They become "legal" because Corp gets 3 2-point agendas from Campaign and Runner gets 3 ressources from campaign

translation of spoiled german rulebook time ago : https://pastebin.com/0eBh6JQm

**Comment by AdrianHerber** [comment]
Since that Agenda and Resource are both provided to the players immediately upon beginning TD, maybe we should make an exception for them (from the rule to not include secret campaign cards) and add them to NRDB? Just adding these two cards would be enough to allow people to create TD decks ready to use at the beginning of the campaign. 

However, we'd (probably?) need to mark them as not legal in any tournament/standard decks. I'm not sure how much of a complication that is - does NRDB already have a flag for that, eg for use on draft cards?

The other restrictions are possible to set up in NRDB deckbuilder already: 
- Legality can be switched to 'casual play'
- Collection can untick packs other than core and TD
- Settings can be set to 'Use 1 Core Set'

**Comment by alsciende** [comment]
Yes, I guess we can add those starting cards, in order for users to be able to build legal decks.

Let's let people build and publish decks with them. At the moment, there's nothing to prevent people to publish "MWL 1.2" decklists with Draft identities, for example. But I agree that it would be nice to have that feature.

**Comment by alsciende** [comment]
I'm going to add a new pack named "TD Campaign" to include those 2 cards, and later, maybe, all the cards from the TD campaign.

**Comment by ANR-PeterPan** [comment]
perhaps make TD a "Cycle" with
1. Tournament legal Cards (perhaps name it Terminal Directive too because of decklists)
2. TD First Game cards
3. TD Campaign cards (Spoiler!)
